test/performance/odp_packet_gen.c

```diff
@@ -560,6 +591,11 @@ static int parse_options(int argc, char *argv[], test_global_t *global)
 		ret = -1;
 	}
 
+	if (test_options->calc_latency && test_options->use_refs) {
+		printf("Error: Latency calculations are not supported with packet references\n");
```

Comment:
`ODPH_ERR()` could be used.

Reply:
`printf()` was the used style. Added a new patch in v2 where all errors/warnings are changed to `ODPH_ERR()`.